src/azul/terra.py

```diff
@@ -513,7 +513,13 @@ def _list_snapshots(self) -> JSONs:
         response = self._check_response(endpoint,
                                         params=dict(limit=limit))
         require(response['total'] <= limit, 'Too many snapshots', response['total'])
-        return response['items']
+        return [
+            {
+                'source_id': snapshot['id'],
+                'source_name': snapshot['name']
+            }
+            for snapshot in response['items']
+        ]
```

Comment:
Why this change?

Reply:
For comparison, this is what an element of `response['items']` looks like:

```
{
      "id": "9449a9f4-85aa-45cb-9fbf-cead88bf8f7d",
      "name": "hca_dev_20201203___20210219",
      "description": "Create snapshot hca_dev_20201203___20210219",
      "createdDate": "2021-03-03T20:17:01.095509Z",
      "profileId": "390e7a85-d47f-4531-b612-165fc977d3bd",
      "storage": [
        {
          "region": "us",
          "cloudResource": "bigquery",
          "cloudPlatform": "gcp"
        },
        {
          "region": "us-central1",
          "cloudResource": "bucket",
          "cloudPlatform": "gcp"
        },
        {
          "region": "us-central1",
          "cloudResource": "firestore",
          "cloudPlatform": "gcp"
        }
      ]
    },
```

Previously the cached snapshots were only used internally. Now that they're exposed to users via `/repository/sources` I decided to put more thought into how the JSON structure should be organized/presented. The main motivation for this change was to present the fields in a way that felt appropriate for the endpoint, but it also has the benefit of reducing our DynamoDB storage footprint.